common/djangoapps/student/forms.py

```diff
@@ -115,12 +115,11 @@ class AccountCreationForm(forms.Form):
     validation, not rendering.
     """
     # TODO: Resolve repetition
-    username = forms.SlugField(
+    username = forms.CharField(
```

Comment:
The manual validation code that this form replaced used django.core.validators.validate_slug, so it became a SlugField in this form.